panel/models/html.ts

```diff
@@ -17,6 +17,7 @@ export class DOMEvent extends ModelEvent {
 
   static {
     this.prototype.event_name = "dom_event"
+    this.prototype.publish = true
```

Comment:
Setting `publish` shouldn't be needed since https://github.com/bokeh/bokeh/pull/12871.

Reply:
Yep, I had just made a copy-paste error in defining one of the event names making me think the publish default wasn't working.